examples/update-demo/README.md

```diff
@@ -26,21 +26,13 @@ $ cd kubernetes
 $ hack/dev-build-and-up.sh
 ```
 
-If you are running your cluster on GCE (the default), you may need to open the firewall for port 8080 using the [console](https://console.developer.google.com) or the `gcloud` tool. The following command will allow traffic from any source to instances tagged `kubernetes-minion`:
-
-```bash
-$ gcloud compute firewall-rules create \
-  --allow tcp:8080 --target-tags=kubernetes-minion \
-  kubernetes-minion-8080
-```
-
 ### Step One: Turn up the UX for the demo
 
 You can use bash job control to run this in the background.  This can sometimes spew to the output so you could also run it in a different terminal.
 
```

Comment:
It's worth mentioning that 8001 is hard coded in the example's local webserver ([index.html](https://github.com/roberthbailey/kubernetes/blob/update-sh/examples/update-demo/local/index.html#L25))